R/pkg/R/DataFrame.R

```diff
@@ -1869,6 +1869,7 @@ setMethod("where",
 #' path <- "path/to/file.json"
 #' df <- read.json(path)
 #' dropDuplicates(df)
+#' dropDuplicates(df, "col1", "col2")
 #' dropDuplicates(df, c("col1", "col2"))
 #' }
 setMethod("dropDuplicates",
```

Comment:
Actually, I kept the existing `dropDuplicates` since it handles `dropDuplicates(df)` for all columns, too.
Don't we still need two functions if we move the case "c(...)"?